src/services/UserInfoRequestProcessor.ts

```diff
@@ -1,3 +1,5 @@
+/* eslint-disable complexity */
```

Comment:
are these two eslint-disable needed?

Reply:
They are needed to suppress warnings
![image](https://github.com/alphagov/di-ipv-cri-cic-api/assets/40401118/10053844-f948-405d-9b21-f4f57c1ed6ab)